couchbase/json-buckets.yml

```diff
@@ -0,0 +1,1071 @@
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.vb_replica_num[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:avg()
+        type: double_gauge
+        label: vb replica num
+        description: Number of replica vBuckets
+
+      - name: vb.replica.ops.create
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.vb_replica_ops_create[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:sum()
+        type: double_counter
+        label: vb replica ops create
+        description: Number of replica create operations
+
+      - name: vb.replica.ops.update
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.vb_replica_ops_update[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:sum()
+        type: double_counter
+        label: vb replica ops update
+        unit: items
+        description: Number of items updated on replica vBucket for this bucket
+
+      - name: vb.replica.queue.age
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.vb_replica_queue_age[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:avg()
+        type: double_gauge
+        label: vb replica queue age
+        unit: ms
+        description: Sum of disk replica queue item age in milliseconds
+
+      - name: vb.replica.queue.drain
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.vb_replica_queue_drain[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:sum()
+        type: double_counter
+        label: vb replica queue drain
+        description: Total drained replica items in the queue
+
+      - name: vb.replica.queue.fill
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.vb_replica_queue_fill[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:sum()
+        type: double_counter
+        label: vb replica queue fill
+        description: Total enqueued replica items in disk queue
+
+      - name: vb.replica.queue.size
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.vb_replica_queue_size[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:avg()
+        type: double_gauge
+        label: vb replica queue size
+        description: Replica items in disk queue
+
+      - name: vb.replica.resident.items.ratio
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.vb_replica_resident_items_ratio[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:avg()
+        type: double_gauge
+        label: vb replica resident items ratio
+        description: Number of resident replica items
+
+      - name: vb_total_queue_age
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.vb_total_queue_age[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:avg()
+        type: double_gauge
+        unit: sec
+        label: vb total queue age
+        description: Sum of disk queue item age in milliseconds
```

Comment:
the unit says seconds, but the description says in milliseconds

Reply:
fixed